### PR TITLE
Support for embed color by perfection for IV alerts

### PR DESCRIPTION
**Pull request by fosJoddie**
Add support to use iv color instead of type color for monster messages

new dts `{{ivcolor}}` will color alarms depending on how good a pokemons IV is:

gray: less than 25 %
white: 25 to 50% 
green: 50 - 82%
blue: 82 to 90 %
purple: 90 to 100%
orange: 100%

### CODE REVIEW

**Comment by Alderon86** [comment]
💯 could be red to match emoji style

**Comment by fosJoddie** [comment]
they follow the world of warcraft loot color, because they are imprinted in many games.

And they follow the rule of "don't let red be best cause Valor would have a field day"

**Comment by Alderon86** [comment]
Valor 4 Life! :D

File: app/src/filter.js [hunk 1]
```diff
@@ -354,3 +356,27 @@ function sendDMAlarm(message, human, e, map) {
         })
     } else log.warn(`Tried to send message to ID ${human}, but didn't find it`);
 }
+
+
+function findRarityColor(iv) {
```
Comment (KartulUdus) [inline app/src/filter.js]: perhaps rarity is misleading, a pidgey can be 💯 but is not rare

{{ivcolor}} perhaps

File: app/src/filter.js [hunk 2]
```diff
@@ -354,3 +356,27 @@ function sendDMAlarm(message, human, e, map) {
...
+    if(!iv || iv < 25) {
```
Comment (KartulUdus) [inline app/src/filter.js]: `!iv` won't happen because missing iv === -1 (%)

File: config/dts.json.example [hunk 3]
```diff
@@ -4,7 +4,7 @@
       "title": "a wild {{{name}}} has appeared!",
       "description": "It weighs {{weight}}kg and will despawn at {{time}}, you have {{tthm}}m {{tths}}s left \nPerfection: **{{iv}}%** , ({{atk}}/{{def}}/{{sta}}) \n{{move1}}|{{move2}} Level:{{level}} (cp:**{{cp}}**)\naddress: {{addr}} ",
       "url": "{{{rocketmap}}}",
-      "color": "{{color}}",
+      "color": "{{rarityColor}}",
```
Comment (KartulUdus) [inline config/dts.json.example]: I like that you can get raritycolor for IV and typecolor for noIV 👍

**Comment by fosJoddie** [comment]
Changed to `ivcolor`

moved colors into the default.json config so people can change color themselves.

Added docs. Heck knows if I made a good table or not

**Pull request closed without merging**